module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -409,10 +431,18 @@ public function handleRequest(Params $params)
         try {
             $results = $this->ils->getStatuses($ids);
         } catch (ILSException $e) {
-            // If the ILS fails, send an empty response instead of a fatal
+            // If the ILS fails, send an error response instead of a fatal
             // error; we don't want to confuse the end user unnecessarily.
             error_log($e->getMessage());
-            $results = [];
+            foreach ($ids as $id) {
+                $results[] = [
+                    [
+                        'id' => $id,
+                        'error' => 'An error has occurred',
+                        'availability_message' => 'status_unknown_message'
```

Comment:
This unknown message is coming through untranslated and not being displayed at all (it remains hidden). Can you make use of `$messages['unknown']` instead to use the appropriately-formatted status?

Reply:
It was a fallback, I don't think it needs to be there at all.